apps/site/src/ideas.css

```
@@ -0,0 +1,1651 @@
+}
+
+.idea-article a {
+  color: var(--accent-2);
+  text-decoration: underline;
+  text-decoration-color: rgba(47, 92, 140, 0.3);
+  text-decoration-thickness: 1px;
+  text-underline-offset: 0.18em;
+}
+
+.idea-article a:hover {
+  text-decoration-color: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)